src/Components/ServiceExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.Framework.DependencyInjection;
+using Microsoft.Framework.DependencyInjection.Extensions;
+using System.Collections.Generic;
+
+namespace Components
+{
+	public static class ServiceExtensions
+	{
+		public static IServiceCollection UseComponentSystem(this IServiceCollection services)
+		{
+			services.TryAdd(DefaultServices());
```

Comment:
Perhaps. This practice was stolen from MS originally:

https://github.com/aspnet/Mvc/blob/27432d94530405f424e9015c8c5311907bd48b3f/src/Microsoft.AspNet.Mvc/MvcServices.cs#L26

However, I think they've changed their mind:

https://github.com/aspnet/Mvc/issues/1634

Trying to work out which pattern is common practice now.